### ISSUE TITLE
Implement gitlens

### ISSUE THREAD

**Issue by wtphilip** [stormkit-io/app-stormkit-io#120]
Implementing Gitlens (https://gitlens.amod.io/) helps new contributers to better understand the Stormkit code.

**Comment by eldemcan** [comment]
@wtphilip can we move this to discussions sections?

**Comment by wtphilip** [comment]
Yes, good idea. I would have to move it to the www-stormkit repository. Or can you, @svedova, create one for this repo as well?
